tests/update_reference_data.sh

```diff
@@ -136,6 +136,7 @@ tests[pinch_multz_all]="flow pinch PINCH_MULTZ_ALL"
 tests[pinch_multz_all_barrier]="flow pinch PINCH_MULTZ_ALL_BARRIER"
 tests[model6_msw]="flow model6 1_MSW_MODEL6"
 tests[norne_reperf]="flow norne NORNE_ATW2013_B1H_RE-PERF"
+tests[compl_smry]="flow cmpl_smry COMPL_SMRY"
```

Comment:
cmpl_smry vs compl_smry

Reply:
Thanks - fixed

Reply:
fairly certain you pushed things here you did not intend however..

Reply:
ehhh - what a mess; thank you ;-)

Extra piggy-back commit removed.